imports/ui/QuestionDisplay.jsx

```diff
@@ -7,63 +7,170 @@
 import React, { Component, PropTypes } from 'react'
 import { $ } from 'jquery'
 import { WysiwygHelper } from '../wysiwyg-helpers'
+import { QUESTION_TYPE } from '../configs'
 
 export class QuestionDisplay extends Component {
 
-  constructor (p) {
-    super(p)
+    constructor (p) {
+        super(p)
 
-    this.readonly = false
-    if (this.props.readonly) this.readonly = this.props.readonly
-  }
+        this.readonly = false
+        if (this.props.readonly) this.readonly = this.props.readonly
 
-  componentDidMount () {
-    this.componentDidUpdate()
-  }
+        this.isQuiz = false
+        if (this.props.isQuiz) this.isQuiz = this.props.isQuiz
+    }
 
-  componentDidUpdate () {
-    MathJax.Hub.Queue(['Typeset', MathJax.Hub])
-  }
+    /* 
+    TODO
+    - props for session mode, results
+    - question types
+    - next/submit button for quiz mode (props onclick to change session state)
 
-  render () {
-    const q = this.props.question
-    return (<div className={'ql-question-display ' + (this.readonly ? '' : 'interactive')}>
+    */
 
-      <div className='ql-question-content'>
-        {WysiwygHelper.htmlDiv(q.content)}
-      </div>
+    componentDidMount () {
+        this.componentDidUpdate()
+    }
 
-      <div className='ql-answers'>
-        {
-          q.answers.map((a) => {
+    componentDidUpdate () {
+        MathJax.Hub.Queue(['Typeset', MathJax.Hub])
+    }
+
+// fix correct colour stuff for states
+    wysiwygContent (answer, content, correct) {
+        let classContent = correct ? 'ql-wysiwyg-content correct-color' : 'ql-wysiwyg-content incorrect-color'
+        return (
+            <div className={classContent}
+            key={'answer_' + answer} >
+                { WysiwygHelper.htmlDiv(content) }
+            </div>)
+    }
+
+    commonContent (typeStr, answer, content, correct) {
+        return (
+            <div className={'ql-' + typeStr + '-content'} key={'answer_' + answer}>
+                <span className={correct ? 'correct-color' : 'incorrect-color'}>{content}</span>
```

Comment:
you shouldn't set the color of the item here based on correct.
a.correct stores whether or not that multiple choice answer is correct or not, so if you set the green/red colour based on that, people will know which option is the correct one. 

previously I was using those colors to style the True/False buttons. You can probably just take those colors out for now. @adamperron will style the options later